src/styles/blocks/header.scss

```
@@ -0,0 +1,66 @@
+.header {
+  padding-right: 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  &__logo {
+    display: block;
+    position: relative;
+    padding-left: 50px;
+    height: 40px;
+  }
+}
+
+.nav {
+  flex-shrink: 0;
+
+  &__menu {
+    display: flex;
+    flex-shrink: 0;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__link {
+    display: inline-block;
+    position: relative;
+    padding: 23px 0;
+    @extend %text-medium;
+    line-height: 14px;
+    text-transform: uppercase;
+    color: $main-accent-color;
+    text-decoration: none;
+  }
+
+  &__list {
+    padding-right: 20px;
+    list-style: none;
+  }
+
+  &__list:last-child {
+    padding-right: 0;
+  }
+
+  &__link:hover {
```

use nested for last-child and hover

```suggestion
  &__list {
    padding-right: 20px;
    list-style: none;
    
    &:hover {
    }
    
    &:last-child {
    }
  }
```